src/main/java/picard/analysis/directed/HsMetricCollector.java

```diff
@@ -109,7 +121,7 @@ public HsMetrics convertMetric(final TargetMetrics targetMetrics) {
      * @return the hs penalty - a multiplier that tells if you want, e.g. 20X coverage, then you will
      *         need to produce this many PF aligned bases per target bases in your design!
      */
-    private double calculateHsPenalty(final Long librarySize, final TargetMetrics targetMetrics, final int coverageGoal) {
+    protected double calculateHsPenalty(final Long librarySize, final TargetMetrics targetMetrics, final int coverageGoal) {
```

Comment:
If you widen the visibility of this method, it would be good to fix the javadoc so it describes all the params.

Reply:
ok, will do